nextflow.config

```diff
@@ -51,19 +93,40 @@ try {
 }
 
 profiles {
-  awsbatch { includeConfig 'conf/awsbatch.config' }
   conda { process.conda = "$baseDir/environment.yml" }
   debug { process.beforeScript = 'echo $HOSTNAME' }
-  docker { docker.enabled = true }
-  singularity { singularity.enabled = true }
+  docker {
+    docker.enabled = true
+    // Avoid this error:
+    //   WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
+    // Testing this in nf-core after discussion here https://github.com/nf-core/tools/pull/351
+    // once this is established and works well, nextflow might implement this behavior as new default.
+    docker.runOptions = '-u \$(id -u):\$(id -g)'
+  }
+  singularity {
+    singularity.enabled = true
+    singularity.autoMounts = true
+  }
   test { includeConfig 'conf/test.config' }
+  galaxy {
+    includeConfig 'conf/base.config'
```

Comment:
You 
```suggestion

```

Reply:
You have already included this above. What does `-profile galaxy` allow you to do? Is it something that should be added to nf-core/configs instead?